services/docker-compose.yml

```diff
@@ -65,9 +65,9 @@ services:
       - STORAGE_PORT=8080
       - CATALOG_HOST=${CATALOG_HOST:-catalog}
       - SWARM_STACK_NAME=${SWARM_STACK_NAME:-simcore}
-      - WEBSERVER_MONITORING_ENABLED=1
       - WEBSERVER_LOGLEVEL=${LOG_LEVEL:-WARNING}
-      - WEBSERVER_DIAGNOSTICS_MAX_DELAY_SECS=30
+      - DIAGNOSTICS_MAX_DELAY_SECS=30
+      - DIAGNOSTICS_MAX_AVG_LATENCY=3
```

Comment:
Cannot be modified through env variable?

Reply:
in this case, there is no need for a double indirection, i.e. environ in host -> environ in docker
We can always change this in portainer to try new values or even osparc-ops allows us to overrride docker-compose and not only .env